ethereum/spec/src/main/java/tech/pegasys/teku/spec/config/SpecConfigBuilder.java

```diff
@@ -176,8 +180,13 @@ public SpecConfig build() {
         bellatrixConfig = Optional.of(bellatrixBuilder.get().build(altairConfig));
         config = bellatrixConfig.get();
       }
+      Optional<SpecConfigCapella> capellaConfig = Optional.empty();
       if (capellaBuilder.isPresent() && capellaBuilder.get().isCapellaIncluded()) {
-        config = capellaBuilder.get().build(bellatrixConfig.orElseThrow());
+        capellaConfig = Optional.of(capellaBuilder.get().build(bellatrixConfig.orElseThrow()));
```

Comment:
Starting from capella we changed the pattern here. We were nesting the altair and bellatrix but then the other are all under altair. Side effect could be that we load config for a fork (ie Capella) without previous fork config being defined (ie Bellatrix).

Reply:
How about this? https://github.com/ConsenSys/teku/pull/6402/commits/dd11f012ed056d891f728efb89fe27a108e8cef5